js-core/homeworks/homework-14/index.js

```diff
@@ -0,0 +1,123 @@
+//TASK 0
+//let solution = [12, 23, 34, 12, 12, 23, 12, 45];
+let solution = [4, 4, 100, 5000, 4, 4, 4, 4, 4, 100, 100,];
+//let solution = [3, 3, 4, 6, 4, 5, 9, 9, 21, 9];
+//let solution = [4, 8, 15, 16, 23, 42, 4, 15, 42, 42];
+//let solution = [2, 2, 44, 44];
+
+function result(arr) {
+	let resArr = [];
+	let objNumbers = arr.reduce((done, elem) => {
+		if(elem in done) {
+			done[elem] += 1;
+		} else {
+			done[elem] = 1;
+		};
+		return done;		
+	}, {});
+	for(let key in objNumbers) {
+		if(objNumbers[key] % 2 != 0) {
+			resArr.push(+key);
+		};
+	};	
+	return resArr;
+};
+console.log(result(solution));
+
+
+
+
+//TASK - 1
+const someWebpackModule = `module.exports = {
+    context: %%HOMEDIR%%,
+    entry: {
+        app: "%%HOMEDIR%%/%%APP_DIR%%/%%APPNAME%%.js"
+    },
+    output: {
+        path: %%HOMEDIR%% + '/app',
+        filename: "dist/[%%APPNAME%%].js",
+        library: '[%%APPNAME%%]'
+    }
+   }`;
+
+//TASK - 1
+let regExpSetpoint = /%%HOMEDIR%%/gmi;
+//let regExpSetpoint = /%%APP_DIR%%/gmi;
+//let regExpSetpoint = /%%APPNAME%%/gmi;
+
+let regExpRequiredValue = './JavaScript-Basic';
+//let regExpRequiredValue = 'fixtures/src';
+//let regExpRequiredValue = 'app.js';
+
+function regExpEndParse(str, setpoints, requiredValue) {
+	let test = str.replace(setpoints, requiredValue);
+	return test;
+}
+
+console.log(regExpEndParse(someWebpackModule, regExpSetpoint, regExpRequiredValue));
+
+
+
+
+//Task3
+window.onload = function() {
```

Comment:
it's not really necessary to start that script onload, do you have any issues if you just execute it normally?